scalableminds/webknossos
# Ensure that segments are added to segment list when using quick select...

… or flood fill to create new segments

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- use quick select and floodfill to create new segments --> should be added to segment list
- floodfill can be tested best by using it to relabel an existing segment (and 2D mode)

### Issues:
- fixes #6858

------
(Please delete unneeded items, merge only when none are left open)
